pkg/api/v1alpha1/vspheredatacenterconfig_webhook.go

```diff
@@ -63,7 +63,8 @@ func (r *VSphereDatacenterConfig) ValidateCreate() error {
 	if !features.IsActive(features.FullLifecycleAPI()) {
 		return apierrors.NewBadRequest("Creating new VSphereDatacenterConfig on existing cluster is not supported")
 	}
-	return nil
+
+	return r.Validate()
```

Comment:
should this run at the beginning as well? I would expect this validations to run even when pause annotation is present